src/firefly/js/visualize/rawData/RawDataOps.js

```diff
@@ -275,40 +279,48 @@ export async function loadStretchData(pv, plot, dispatcher) {
     const oPv= mask ? getOverlayById(pv,imageOverlayId) : undefined;
     const maskOptions= mask ? {maskColor:oPv?.colorAttributes.color, maskBits: oPv?.maskValue } : undefined;
     const dataCompress= getFirstDataCompress(plot,mask);
-    const dataSize= plot.dataWidth*plot.dataHeight;
     const {success:firstSuccess, fatal}= await loadStandardStretchData(workerKey, plot,
                   {dataCompress, backgroundUpdate:false, checkForPlotUpdate:!mask}, maskOptions);
 
     if (plotInvalid()) return;
     if (firstSuccess) {
         dispatcher({ type: ImagePlotCntlr.BYTE_DATA_REFRESH, payload:{plotId, imageOverlayId, plotImageId}});
-        if (dataCompress==='FULL') return;
+        if (dataCompress===FULL) return;
```

Comment:
Looks like regardless of the condition, it will 'return' because there's nothing else to do after it leaves the 'if' block.

Reply:
yes, I can remove the line, intellij is tell me the same thing.